components/02-elements/dropdown-list/_dropdown-list.scss

```diff
@@ -47,6 +50,7 @@ $dropdown-list__item-color--inner         : $white !default;
 $dropdown-list__icon-fill--inner          : $white !default;
 $dropdown-list__item-border--inner        : 1px solid $white !default;
 $dropdown-list__item-border--inner-level2 : 1px solid $gray-dark !default;
+$dropdown-list__item-font-weight--inner   : $font-weight-base !default;
```

Comment:
Use `$spacer` variables where possible instead of number values (for 16px, maybe it will be good to add $spacer var for 24px? it's also multiplication of 8). What do you think?

Reply:
Yes and no, because right now we have four variants of spacer:
```scss
$spacer                     : 8px !default;
$spacer--medium             : 16px !default;
$spacer--large              : 32px !default;
$spacer--extra-large        : 48px !default; 
```

I don't have idea how to name spacer between `--medium` and `--large`. We can add `--small` spacer and move`--medium` to 24px, but it require to change this variable in the whole alpaca project..
```scss
$spacer                     : 8px !default;
$spacer--small             : 16px !default;
$spacer--medium             : 24px !default;
$spacer--large              : 32px !default;
$spacer--extra-large        : 48px !default; 
```

Reply:
ok, I agree about changing spacer names b/c they are used in multiple places, but could you please add an issue for additional variable with 24 with some goo name between medium and large... I would like others think about it and decide. Thnx